roles/install-docker-ce/tasks/main.yml

```diff
@@ -33,25 +33,26 @@
           apt-get install docker-ce=$(apt-cache madison docker-ce |grep "{{ docker_ce_version }}" | head -n 1 | awk '{print $3}') -y
       fi
 
-
-      # Setup daemon.
-      cat > /etc/docker/daemon.json <<EOF
-      {
-        "exec-opts": ["native.cgroupdriver={{ docker_vcgroupdriver }}"],
-        "log-driver": "json-file",
-        "log-opts": {
-          "max-size": "100m"
-        },
-        "storage-driver": "overlay2"
-      }
-      EOF
-
-      mkdir -p /etc/systemd/system/docker.service.d
-
-      iptables -F
-      # Restart docker.
-      systemctl daemon-reload
-      systemctl restart docker
+      if [[ "{{ arch }}" == "amd64" ]]; then
+        # Setup daemon.
+        cat > /etc/docker/daemon.json <<EOF
+        {
+          "exec-opts": ["native.cgroupdriver={{ docker_vcgroupdriver }}"],
```

Comment:
ARM don't support cgroup option now, so please only disable `"exec-opts": ["native.cgroupdriver={{ docker_vcgroupdriver }}"],` in daemon.json.